### PR TITLE
Project description

### PR DESCRIPTION
**Pull request by johnjohndoe**
- I added a README.md. Please review and feel free to modify or extend it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+A maintenance script for the [Wo ist Markt? project](https://github.com/wo-ist-markt/wo-ist-markt.github.io)
+to automatically check if the market data for Berlin has changed.
+
+
+## Project description
+
+The script compares two time stamps. One can be found on the website for
+the Berlin market data on the data portal. The counterpart can be found at the
+bottom of the JSON file which is rendered at the website.
```
Comment (k-nut) [inline README.md]: which website is the last sentence referring to? Also maybe it makes sense to add links to both sources?

Comment (johnjohndoe) [reply README.md]: I was not sure if it is smart to duplicate the links as the script already hardcodes them.

Comment (k-nut) [reply README.md]: I think it would be better than to tell people to read the source.

Comment (johnjohndoe) [reply README.md]: @k-nut Fixed in 280ccd1c62130f7d54b59995ad47a7ef3e28e824.

**Review by k-nut — APPROVED** [review]

**Pull request merged**